src/main/scala/org/hammerlab/guacamole/pileup/PileupElement.scala

```diff
@@ -216,7 +216,7 @@ case class PileupElement(
    */
   @tailrec
   final def advanceToLocus(newLocus: Long): PileupElement = {
-    assume(newLocus >= locus, "Can't rewind to locus %d from %d. Pileups only advance.".format(newLocus, locus))
+    assume(newLocus >= locus, s"Can't rewind to locus $newLocus from $locus. Pileups only advance. $read")
```

Comment:
I think I added that just to help track down situations where this error state was hit, while debugging things. NBD either way if you want to remove it.